### PR TITLE
KT-32640 KDoc. Support markdown link in '@see' block

### PR DESCRIPTION
**Pull request by turansky**
Related issue - [KT-32640](https://youtrack.jetbrains.com/issue/KT-32640)

Screenshot 2020-04-23 at 18 52 19

### CODE REVIEW

File: idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt [hunk 1]
```diff
@@ -199,6 +198,18 @@ object KDocRenderer {
         }
     }
 
+    private fun markdownToHyperLink(markdown: String): String? =
```
Comment (darthorimar) [inline idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt:201]: Do we to need to check the markdown structure here? Maybe just `parseMarkdown(markdown).toHtml()`?

**Comment by vladimirdolzhenko** [comment]
Any update on this ?

File: idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt (hunk 1 above)
Comment (turansky) [reply idea/src/org/jetbrains/kotlin/idea/kdoc/KDocRenderer.kt:201]: I think that `@see` block must support only links, other markdown looks like error

**Comment by turansky** [comment]
Not soon. Feel free to use these changes for new PR (with tests)

**Pull request closed without merging**